tests/gocase/unit/type/strings/strings_test.go

```diff
@@ -497,7 +573,12 @@ func TestString(t *testing.T) {
 	})
 
 	t.Run("Extended SET with incorrect use of multi options should result in syntax err", func(t *testing.T) {
-		require.ErrorContains(t, rdb.Do(ctx, "SET", "foo", "bar", "ex", "10", "px", "10000").Err(), "syntax err")
```

Comment:
You can still retain this case? Although it's semantically equal to `px 100 ex 10`, we're testing the parsing logic here so `ex 10 px 10000` can be a different case.

Reply:
OK. Tests for different parameter orders have been added.